Example/Messaging/Sample/Sample/ContentView.swift

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import FirebaseCore
+import FirebaseMessaging
+import FirebaseInstanceID
+import FirebaseInstallations
+
+struct ContentView: View {
+  @EnvironmentObject var identity: Identity
+  
+  var body: some View {
+    NavigationView {
+      List {
+        Button(action: getToken) {
+          HStack{
```

Comment:
nit: Space between `HStack` and `{` (throughout file)

Reply:
Done. Sorry forgot to run clang/format